app/scripts/contentscript.js

```diff
@@ -181,12 +181,12 @@ function documentElementCheck () {
 }
 
 /**
- * Checks if the current domain is blacklisted
+ * Checks if the current domain is blocklisted
```

Comment:
Nit: I think we should use "blocked" here instead of "blocklisted"

```suggestion
 * Checks if the current domain is blocked
```

Reply:
It looks like there are a bunch of other examples of this, e.g. `blocklistedDomainCheck`. Though "blocked` is already used in some places.

Reply:
do you think we should change it to blockedDomainCheck? Or are you referring to in the comments?

Reply:
Yeah, `blockedDomainCheck` would sound better. I've never seen the verb `blocklisted` used before, usually it's just `blocklist` the noun, and the verb is `blocked`.